src/LineBot/LineBot.cs

```diff
@@ -69,20 +69,20 @@ internal LineBot(ILineConfiguration configuration, HttpClient client, ILineBotLo
         /// <summary>
         /// Creates a rich menu.
         /// </summary>
-        /// <param name="richMenu">The rich menu represented as a rich menu object.</param>
+        /// <param name="richMenuRequest">The rich menu represented as a rich menu object.</param>
         /// <returns>.</returns>
-        public async Task<string> CreateRichMenu(RichMenu richMenu)
+        public async Task<string> CreateRichMenuRequest(IRichMenuRequest richMenuRequest)
```

Comment:
No need to rename this method. Better to keep it called `CreateRichMenu`. And I just noticed that in the API documention `RichMenu` is used for the request and `RichMenuResponse` for all the answers.  Still not happy with the naming and sorry for making you change this but I think I want it back with the names that you used before.

Reply:
Btw, the response they give, only include an id.